features/refund_error_recovery.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Refund form with error recovery
+  Background:
+    Given I am "Anakin Skywalker"
+    # No town/city, county, email address in applicant page
```

Comment:
Can we remove the commented out lines?

Reply:
We can, but I wanted them there to remind me what is behind 'Anakin Skywalker' - i.e. what is different about him